jinja2/includes/newsletter.html

```diff
@@ -1,31 +1,30 @@
 <div class="newsletter">
     <form id="newsletterForm" class="newsletter-form nodisable" name="newsletter-form" action="https://www.mozilla.org/en-US/newsletter/" method="post">
         <div class="newsletter-head">
-            <h2 class="newsletter-teaser">{{ _('Learn the best of web development') }}</h2>
-            <p class="newsletter-description">{{ _('Get the latest and greatest from MDN delivered straight to your inbox.') }}</p>
+            <h2 class="newsletter-teaser">{{ _('Learn the best of web development. Get the latest and greatest from MDN delivered to your inbox.') }}</h2>
+            <div id="newsletterPrivacy" class="form-group form-group-agree newsletter-group-privacy">
+                <input type="checkbox" id="newsletterPrivacyInput" name="privacy" required>
+                <label for="newsletterPrivacyInput">
+                {% trans policy_url="https://www.mozilla.org/privacy/" %}
+                    I'm okay with Mozilla handling my info as explained in this <a href="{{ policy_url }}">Privacy Policy</a>.
+                {% endtrans %}
+                </label>
+            </div>
             {% if request.LANGUAGE_CODE != 'en-US' %}
-            <p class="newsletter-lang">{{ _('The newsletter is offered in English only at the moment.') }}</p>
+                <p class="newsletter-lang">{{ _('The newsletter is offered in English only at the moment.') }}</p>
             {% endif %}
         </div>
 
         <div class="newsletter-fields">
             <input type="hidden" id="fmt" name="fmt" value="H">
             <input type="hidden" id="newsletterNewslettersInput" name="newsletters" value="app-dev" />
-            <div id="newsletterErrors" class="newsletter-errors"></div>
 
             <div id="newsletterEmail" class="form-group newsletter-group-email">
                 <label for="newsletterEmailInput" class="form-label offscreen">{{ _('E-mail') }}</label>
                 <input type="email" id="newsletterEmailInput" name="email" class="form-input newsletter-input-email" required placeholder="{{ _('you@example.com') }}" size="30">
```

Comment:
Not introduced here but I noticed that this `input` has a `size` attribute with a limit of 30. For consistency, we should remove it here. Also, some people have really looooong email addresses especially if they use something like myname+mdnwebdocs@gmail.com

Reply:
Removed. Good catch!